docs-md/development/jemalloc.md

```diff
@@ -32,6 +32,20 @@ This will turn on profiling from that moment until you disable it via:
 
 In a variety of cases, it might be desirable to have profiling active from the point
 of application start.  To do this set `prof_active:true` in the `MALLOC_CONF`.
+However keep in mind that there is some performance cost while profiling is active,
+so you may only want to keep profiling active when you are gathering memory usage
+information.  And you may also want to adjust the sampling if performance is adversely
+affected (see the jemalloc link below if this is a concern).
+
+NOTE: It is a good idea to confirm these settings changes and check status (at any
+time) by simply curling:
+
+`/mtev/heap_profile`
```

Comment:
```suggestion
`yourmachine:yourport/mtev/heap_profile`
```

Reply:
For consistency with the curl example in the next section.

Reply:
fixed